htmresearch/frameworks/nlp/imbu.py

```diff
@@ -145,6 +147,11 @@ def _modelFactory(self, modelType, **kwargs):
                     prepData=False,
                     retinaScaling=1.0)
 
+    elif modelType == "Keywords":
+      # k should be the number of data samples (i.e. numLabels) because Keywords
```

Comment:
This logic is from the initial writing of the querying method, where we couldn't get distance data out of the KNNClassifier region, so I classified each document as it's own category, and then used the categoryVotes for query results. It's confusing and no longer necessary, so I'm fixing it in the next push.